src/App.tsx

```diff
@@ -54,6 +56,8 @@ export enum Routes {
   CHANGE_ACCOUNT_EMAIL = '/change-email',
   EVENT_REGISTRATIONS = '/events/:id/rsvp',
   FAMILY_DETAILS = '/family-details/:id',
+  VIEW_REQUESTS = '/view-requests',
+  VIEW_PF_REQUEST = '/view-request/:request_id/:user_id',
```

Comment:
I feel like this could be simplified to surface the most recent request for the user, so we don't have to rely upon request_id